src/screens/discover/components/DiscoverSearch.js

```diff
@@ -12,13 +12,13 @@ import { InteractionManager, View } from 'react-native';
 import { IS_TESTING } from 'react-native-dotenv';
 import { useDispatch } from 'react-redux';
 import { useDebounce } from 'use-debounce/lib';
-import CurrencySelectionTypes from '../../helpers/currencySelectionTypes';
-import { emitAssetRequest } from '../../redux/explorer';
-import deviceUtils from '../../utils/deviceUtils';
-import { CurrencySelectionList } from '../exchange';
-import { initialChartExpandedStateSheetHeight } from '../expanded-state/asset/ChartExpandedState';
-import { Row } from '../layout';
-import DiscoverSheetContext from './DiscoverSheetContext';
+import CurrencySelectionTypes from '@rainbow-me/helpers/currencySelectionTypes';
```

Comment:
Nit: use the `@/` prefix. Maybe this was your IDE?

Reply:
Most likely